docs/OV_Runtime_UG/integrate_with_your_application.md

```diff
@@ -246,18 +248,15 @@ The example uses CMake for project configuration.
 
```

Comment:
The heading should be "Linking and Building Your C++ Application with OpenVINO™ Runtime"

Reply:
Applied in https://github.com/openvinotoolkit/openvino/pull/11658/commits/c3f936da7112ecba3bb036be060690677dcfc290